src/api/app/models/branch_package.rb

```diff
@@ -273,12 +273,13 @@ def create_branch_project
 
       title = "Branch project for package #{params[:package]}"
       description = "This project was created for package #{params[:package]} via attribute #{@attribute}"
+      base_project_url = Project.find_by_name(params[:project]).try(:url)
```

Comment:
is there a chance that params[:project] will not be received, also if there is a `params[:request]` should there be any changes?

Reply:
Looking into the code base, right now `params[:project]` is always being passed but there is nothing in the code that prevents anyone from not passing it.

I don't know what to say about your second question... Maybe someone else can shed a bit of light?

Reply:
I guess then its a misunderstanding from me, I think than its ready for a merge, unless there is any other comments

Reply:
The presence of `params[:request]` shouldn't affect the URL, so the changes are fine.